crypto/bn/asm/rsaz-2k-avx512.pl

```diff
@@ -546,36 +542,36 @@ sub amm52x20_x1_norm {
     vpbroadcastq    $red_tbl_idx, $idx
```

Comment:
line 538
why not use method used for 3k, which is a little easier to read
    movq    $tbl_idx, %rax
    imulq   \$`20*8`, %rax
    addq    %rax, $red_tbl

Reply:
comment no longer relevant with new implementation